docs/metrics/metric-rules-editor.md

```diff
@@ -82,7 +88,8 @@ Different classes have dot-delimited names of various lengths, so a match expres
 
 Here's how to create a metrics rule using the metrics rules editor in the Sumo UI:
 
-1. In the Sumo web app, go to **Manage Data > Metrics > Metrics Rules**. The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Rules**. <!--Kanso 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Metrics Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Rules**. Kanso--><br/>The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
```

Comment:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics**, select **Metrics Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Rules**. Kanso--><br/>The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
```

Reply:
I'll leave this as-is since it's consistent with many other entries.